res/schema.xml

```diff
@@ -447,4 +447,37 @@ METADATA
       ALTER TABLE cues ADD COLUMN source INTEGER DEFAULT 2 NOT NULL;
     </sql>
   </revision>
+  <revision version="30" min_compatible="3">
+    <description>
+      Tables for Rekordbox library feature
+    </description>
+    <sql>
+     CREATE TABLE IF NOT EXISTS rekordbox_library (
```

Comment:
Readability could be improved by following consistent rules for upper-/lowercase writing, i.e. UPPERCASE for all SQL(ite) keywords and snake_case for table/column names.